### DISCUSSION TITLE
Meeting Agenda 2/20

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#293] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Discuss project velocity
3. Sprint Planning
4. Finish the Lean Canvas https://docs.google.com/presentation/d/1Y9nodhgXYgMNK-OpEW6yPCJz_zJHcsCT-Kng8YCzfKk/edit#slide=id.p

Note: Mike will be absent

**Comment by Victor-M-Giraldo** [comment]
Meeting minutes 2/20

1. Task Review
- Got rid of a bunch of unused dependencies.
- Upgraded certain dependencies
- Pending upgrade from Yarn 1 to Yarn 4 (Might break dev environments)
- Added average load test that better approximates average peak load of system in production.
- Fixed OTT-Common import
- Completed [#1297](https://github.com/dyc3/opentogethertube/issues/1297)
- Completed [#1252](https://github.com/dyc3/opentogethertube/issues/1252)
- Completed [#1286](https://github.com/dyc3/opentogethertube/issues/1286)
- Completed [#1320](https://github.com/dyc3/opentogethertube/issues/1320)
- Delayed [#1206](https://github.com/dyc3/opentogethertube/issues/1206)
- Delayed #256 
- Delayed #276 
- Finished Lean Canvas

3. Sprint Planning
- Pick up pace regarding visualization.
- Ideally get a MVP done within two sprints.
- Assigned [#1339](https://github.com/dyc3/opentogethertube/issues/1339) to Victor.
- Assigned [#1358](https://github.com/dyc3/opentogethertube/issues/1358) to Chris.
- Assigned [#1357](https://github.com/dyc3/opentogethertube/issues/1357) to Carson.
- Assigned [#1359](https://github.com/dyc3/opentogethertube/issues/1359) to Carson.
- Assigned [#1343](https://github.com/dyc3/opentogethertube/issues/1343) to Carson.
- Assigned [#1360](https://github.com/dyc3/opentogethertube/issues/1360) to Mike.
